test/fixtures/autofix/ambiguousEventHandlers/Main.view.xml

```diff
@@ -0,0 +1,43 @@
+<mvc:View xmlns="sap.m" xmlns:mvc="sap.ui.core.mvc" xmlns:core="sap.ui.core"
+	controllerName="ui5.walkthrough.controller.Main">
```

Comment:
We should also add a test case without a `controllerName`. In that case, we can't provide any fixes.

Reply:
Thanks!